test/real_content/swupd_update_fullfiles.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load "real_content_lib"
+
+# Test massive fullfile downloads
+@test "RC004: Update from first to last with --fix" {
+	# shellcheck disable=SC2153
+	echo "Install minimal system with oldest version (${VERSION[0]})"
```

Comment:
all these messages that you are printing with "echo" won't show on the test's output unless the test fails.
I would suggest to use the _print_ function provided by the test library so you can see those messages as part of the output when using _bats -t_ to run a test.

Reply:
Thanks. I'll fix that